DQMOffline/JetMET/src/JetAnalyzer.cc

```diff
@@ -1974,12 +2159,15 @@ void JetAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetu
       puidmvaflag=(*puJetIdFlagMva)[pfjetref];
       puidcutflag=(*puJetIdFlag)[pfjetref];
       jetpassid = pfjetIDFunctor((*pfJets)[ijet]);
-      if((*pfJets)[ijet].muonEnergyFraction()>0.8){
+      if(fabs ((*pfJets)[ijet].eta()>3.0)){
```

Comment:
The bracket of fabs needs to be closed after eta().
Would you mind putting this change directly in the pfjetIDFunctor to avoid this duplication?